### ISSUE TITLE
Error: Invalid cell: lacks ref attribute -- cannot read certain xlsx files properly unless file is opened and re-saved first

### ISSUE THREAD

**Issue by donboyd5** [hadley/readxl#102]
Hi,

When I try to use read_excel to read certain .xlsx files that I have downloaded from the internet, I get "Error: Invalid cell: lacks ref attribute," although xlsx:read.xlsx can read them properly. Furthermore, similar to issue #99, if I open the file manually with Excel and save it, the problem goes away. Here is an example:

```
> library(readxl)
> library(xlsx)
> library(dplyr)
> 
> url <- "http://mort.soa.org/Export.aspx?Type=xlsx&TableIdentity=1594"
> fname <- "e:/temp/tabl1594.xlsx"
> 
> download.file(url, fname, mode="wb")
trying URL 'http://mort.soa.org/Export.aspx?Type=xlsx&TableIdentity=1594'
Content type 'application/octet-stream' length 9227 bytes
downloaded 9227 bytes

> 
> df1 <- read.xlsx(fname, sheetIndex = 1) # this works
> glimpse(df1)
Observations: 91
Variables:
$ Table.Name.                                            (fctr) Table Identity:, Provider Domain:, Provider Name:, Table Reference:, Content...
$ RP.2000.Mortality.Table.â...Male.Aggregate...Employees (fctr) 1594, soa.org, Susie Lee, Retirement Plans Experience Committee of the Socie...
> 
> df2 <- read_excel(fname) # Error: Invalid cell: lacks ref attribute
Error: Invalid cell: lacks ref attribute
> glimpse(df2)
Error in nrow(tbl) : object 'df2' not found
> 

> # now manually open file with Excel and save it, then retry:
> df3 <- read_excel(fname)
> glimpse(df3)
Observations: 93
Variables:
$ Table Name:                                          (chr) "Table Identity:", "Provider Domain:", "Provider Name:", "Table Reference:", "C...
$ RP-2000 Mortality Table – Male Aggregate - Employees (chr) "1594", "soa.org", "Susie Lee", "Retirement Plans Experience Committee of the S...
> 

```

Many thanks.

Don

**Comment by jennybc** [comment]
I can reproduce this, but can't immediately lump with one of the non-standard xml problems I already know about.

**Comment by jennybc** [comment]
Will be fixed by #240. Related to #163.
